ibis/backends/tests/test_array.py

```diff
@@ -349,13 +376,27 @@ def test_unnest_default_name(backend):
         (None, None),
         (3, None),
         (-3, None),
-        (None, -3),
         (-3, -1),
+        param(
+            None,
+            -3,
+            marks=[
+                pytest.mark.notyet(
+                    ["flink"],
+                    raises=AssertionError,
+                    reason=(
+                        "DataFrame.iloc[:, 0] (column name='sliced') values"
```

Comment:
When writing `reason`s for `AssertionError`s it's useful to write something other than the assertion error's message, like "Not equivalent due to a bug in Flink" (I'm not saying that's the reason, that's just an example).

Reply:
Thanks, modified the `reason`. It is now hinting at the underlying reason for the failure.